### DISCUSSION TITLE
Chrome browser crashed when cypress tests are running

### DISCUSSION THREAD

**Discussion by sg-ananth** [cypress-io/cypress#27445] [Questions and Help]
HI Team,My Name is ananth
using cypress since two years ,If my scripts are running more than 5 mins Chrome browser crashing with aw- snap error
I used. numTestsKeptInMemory: 0, in cypress.config.js but still same issue my cypress version is 12.12.0 please provide some solution and my browser version is Chrome 115

**Comment by uladzimirdev** [comment]
✅ We ended up sticking to Chrome 111 for running tests which didn't have issues with memory ✅ 

-------------

Same problem, our tests started crashing in chrome 115. Memory usage is not crazy, but for a spec with ~20 tests on the page memory usage grows. Anyhow page can crash on second test when browser monitor shows only ~1-1.5Gb used.
It happens locally and at Github Actions, we had to downgrade chrome to 114 as a workaround and stumbled upon another issue about CRI reconnection to devtools. 

I can provide logs etc, but so far I don't see anything special neither in cypress logs nor in resources that browser uses.

We tried `numTestsKeptInMemory`, experimental `experimentalMemoryManagement` 

Cypress version is `12.17.3`

**Comment by silvio-romero-intelex** [comment]
Same problem here
![image](https://github.com/cypress-io/cypress/assets/147631483/747b8820-8e5f-403a-b917-4f270d11800c)
